resources/views/Admin/include/footerJs.blade.php

```diff
@@ -0,0 +1,52 @@
+
+<!-- plugins:js -->
+
+ <script src="{{asset('admin/vendors/js/vendor.bundle.base.js')}}"></script>
+ <script src="{{asset('admin/vendors/bootstrap-datepicker/bootstrap-datepicker.min.js')}}"></script>
+
+ <!-- endinject -->
+ <script src="{{asset('admin/vendors/typeahead.js/typeahead.bundle.min.js')}}"></script>
+ <script src="{{asset('admin/vendors/select2/select2.min.js')}}"></script>
+
+ <!-- Plugin js for this page -->
+ <script src="{{asset('admin/vendors/chart.js/Chart.min.js')}}"></script>
+ <script src="{{asset('admin/vendors/progressbar.js/progressbar.min.js')}}"></script>
+ <!-- End plugin js for this page -->
+ <!-- inject:js -->
+ <script src="{{asset('admin/js/off-canvas.js')}}"></script>
+ <script src="{{asset('admin/js/hoverable-collapse.js')}}"></script>
+ <script src="{{asset('admin/js/template.js')}}"></script>
+ <script src="{{asset('admin/js/settings.js')}}"></script>
+ <script src="{{asset('admin/js/todolist.js')}}"></script>
+
+ <!-- endinject -->
+ 
+ <!-- Custom js for this page-->
+ <script src="{{asset('admin/js/dashboard.js')}}"></script>
+ <script src="{{asset('admin/js/file-upload.js')}}"></script>
+ <script src="{{asset('admin/js/typeahead.js')}}"></script>
+ <script src="{{asset('admin/js/select2.js')}}"></script>
+
+ <!--Mahitab added this file on 19/3/2024-->
+ <script src="{{asset('admin/js/welcome-txt.js')}}"></script>
+ <script src="{{asset('admin/js/table-pagination.js')}}"></script>
+ {{-- <script src="{{asset('admin/js/edit_form.js')}}"></script> --}}
+ <script src="{{asset('admin/js/add-event.js')}}"></script>
+ <script src="{{asset('admin/js/maps.js')}}"></script>
+ <script src="{{asset('admin/js/edit.js')}}"></script>
+ <script src="{{asset('admin/js/stylingTags.js')}}"></script>
+ <script src="{{asset('admin/js/edit_form.js')}}"></script>
+
+
+ <!-- End custom js for this page-->
+ <script>
```

Comment:
Hello Walaa, this is the only one concern. I just forget to mention it in the meeting. but when you take code from frontend, make sure to paste the code with the same order, as this the last scripting tag must be in top of script as there is another script depends on it.